setup.py

```diff
@@ -2,33 +2,39 @@
 
 import setuptools
 
+with open("README.md", "r", encoding="utf-8") as fh:
+    long_description = fh.read()
+
 requires = ["base58", "binary-helpers", "coincurve", "btclib"]
 
 tests_require = [
-    "flake8>=4.0.1",
-    "flake8-import-order>=0.18.1",
-    "flake8-print>=5.0.0",
-    "flake8-quotes>=3.3.1",
-    "pytest>=3.6.1",
-    "isort>=5.10.1",
-    "pytest-cov>=2.5.1",
-    "black>=22.3.0",
+    "flake8",
```

Comment:
Should we keep these as specific versions?
e.g., one of those packages could  introduce a change that breaks our tests.

Reply:
It's Solar policy to use explicit versions, not "greater than or equal to".

Reply:
*or no version

Reply:
Everyone here probably remembers what happened upstream when `xstate` bumped their minor version with a breaking change because, at that time, upstream's dependencies did not use explicit versions. Suddenly Core stopped working. :)

Reply:
These are just dev requirements, they would only get installed manually if a developer does it when setting his solar development environment. One reason why I removed the version lock is because for example `flake8-print>=5.0.0` is unable to install on Py3.6.

When running eg `pip install solar-crypto` only `requires = ["base58", "binary-helpers", "coincurve", "btclib"]` will get installed. Note that none of those has a fixed version atm. If we wish to change those to a fixed version I'm happy to look into explicit versions might not be possible if we wish to support 3.6 and onwards so in that case we'll need to use `<=>` or something

Usage of Py3 subversions:

![Screenshot from 2022-05-31 18-45-39](https://user-images.githubusercontent.com/60887596/171229093-fa04977f-3bfe-4cce-a117-05fc5004ee4f.png)

Source: https://w3techs.com/technologies/details/pl-python/3

Reply:
Explicit versions are mandatory :)

Reply:
I've never tried, but I believe that would be a possible solution. I've checked and all of our required libraries work with py3.6. Only thing left are the `dev` and `test` requirements.

Reply:
After further research and discussion, it has been decided that test deps will remain pinned and regular deps will **not** explicitly declare a version; as is standard practice in Python libraries as well as in the packages we use.

Resolved in #26